contribution_process.md

```diff
@@ -0,0 +1,149 @@
+during the review process focuses on the implementation of the agreed upon
+design.
+
+Issues that do not receive enough :+1: reactions and do not have any activty for
+a year are considered stale. Without further interaction, they will be closed.
+
+## Contribution-ready stage
+
+A GitHub issue enters the contribution ready stage when a maintainer adds the
+issue to the "Ready to contribute" lane of the public [GitHub project
+board](https://github.com/orgs/aws/projects/21). This stage indicates any user
+is welcome to initiate a contribution via a GitHub pull request.
+
+## Implementation stage
+
+A GitHub issue enters the implementation stage once a user submits a GitHub pull
+request implementing the change. This stage indicates that a contribution has
+been submitted for the issue, but the maintainers have not committed to a full
+review of the contribution. The issue will be moved to the "Implementation" lane
+of the [GitHub project board](https://github.com/orgs/aws/projects/21). The
+maintainers will respond to let the contributor know to expect a preliminary
+review to confirm:
+
+1. It is implemented in the manner described in the issue.
+1. It meets the criteria specified in the contribution guide.
+
+If the criteria are not satisfied, the maintainers will comment on the issue
+specifically indicating what work is still required. If the contributor has
+questions, they should refer to this guide or ask the maintainers to clarify.
+Once the contributor makes the required changes, the maintainers will follow up.
+
+If the contributor is no longer actively engaged by responding to comments from
+maintainers or making code changes for more than 30 business days, the
+contribution is stale. If not, the pull request will be closed and the
+corresponding issue will be marked as ready for contribution.
+
+Contributions made without a GitHub issue are not prioritized to move to the
+next stage. Pull requests that are not linked to an issue and are more than six
+months old are considered stale and will be closed.
+
+## Ready for review stage
+
+Once the preliminary review criteria from the implementation stage have been
+met, the tracking GitHub issue is labeled as ready for review and added to a
+queue for maintainer review. The maintainers select issues for full review
+from this queue. The issues are prioritized for review publicly based on the
+following criteria in descending order of importance:
+
+1. Bugs, documentation changes, and feature requests are prioritized in that
+   order.
+1. Older contributions are selected before more recent ones.
+
+A maintainer must select the issue from the queue with the current highest
+priority. The prioritization of issues in the queue is reviewed on a regular
+cadence and is ultimately decided based on the maintainers' discretion.
+
+## Review stage
+
+Selecting a pull request for review moves the tracking GitHub issue to the
+under review stage. In this stage, the maintainers work with the
```

Comment:
The stage names should potentially be quoted or italicized. It's a little difficult to parse currently.